src/avt/Filters/avtXRayFilter.C

```diff
@@ -151,6 +151,98 @@ inline void Cross(double result[3], const double v1[3], const double v2[3])
     result[2] = (v1[0] * v2[1]) - (v1[1] * v2[0]);
 }
 
+// ------------------------------------------------------------------------- //
+#ifdef PARALLEL
+// ------------------------------------------------------------------------- //
+// these helpers are only needed for MPI case
```

Comment:
What happens if for example some developer introduces the use of an `int` variant of these? Will *all* compilers disallow it without having the subsequent `int` template instantiation? I assume so but don't honestly recall now.

Reply:
The compiler will yell b/c it doesn't have a way to instantiate. I think thats fine.

I thought about moving these into our parallel helpers, but then I would need to cover more cases and I was worried about broader testing. 

 At this point I don't think we use this pattern anywhere else, so I think its ok to leave it local.